### ISSUE TITLE
Support back/next option on wizard flow

### ISSUE THREAD

**Issue by 0xmaayan** [aptos-labs/create-aptos-dapp#14]
When starting the wizard and follow each step, there is no way to go back and/or forward to a step. 

Add back/next option to previous/next step and respect any choice changes.

**Comment by SethY8s** [comment]
@0xmaayan Hey, I haven't looked at the project much yet, but I will give it my best shot and submit a PR sometime today if I finish.

**Comment by SethY8s** [comment]
@0xmaayan I am not super familiar with these terminal prompts, but after looking at the project for a while, I am confident I could add a confirmation prompt at the end of the wizard, something like.

```shell

Project name: 'fakeName',
Choose how to start: ''Toollist Dapp,
Choose network: 'Devnet',
Choose your package manager: 'npm',

Do you wish to change anything? Yes | No. 

``` 

Then if they click yes, it would let them choose which item they want to change, and they could edit the results from there.

**Comment by 0xmaayan** [comment]
@SethY8s Yes I think is it a great addition! how would you let them change an option? would you redirect them back to the step they want to change?

**Comment by mdmudassir0143** [comment]
Or maybe we can give freedom to choose what step they want to modify -
```
Do you wish to change anything? (Yes | No): Yes
Which step would you like to modify? (Step 1, Step 2, Step 3, Step 4): Step 3
Please input a new network: 'Testnet'
```

**Comment by SethY8s** [comment]
@mdmudassir0143 Yes I am working on adding that now.
